r_markdown/benzene_emissionsfactor_qa.Rmd

```diff
@@ -0,0 +1,171 @@
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}  
+  benzene <- dplyr::left_join(benzene, 
+                              dplyr::select(samples, id, fuel, stove, fuelcat, stovecat),
+                              by = "id") %>%
+             dplyr::left_join(dplyr::select(fuel_burnt, - fuel),
+                              by = "id")
+```
+
+* The `benzene` dataframe now contains a subset of the emissions data for only the pollutant benzene. A summary of the dataset is below:  
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  summary(benzene)
+```
+
+## Calculate Emissions Factor
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  benzene <- dplyr::mutate(benzene, pol_fuel_mass_ratio = mass_emitted * 1e-6 / mass_fuel)  # ug * 1e-6 / g
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  summary(benzene$pol_fuel_mass_ratio)
+```
+
+* units: grams of pollutant emitted per gram of fuel burned **NOTE: VERIFY UNITS**
+
+# Plots
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 12}
+ggplot(benzene, aes(x= id, pol_fuel_mass_ratio, colour = fuelcat)) +
+  geom_point() +
+  ggtitle("Benzene EFs by Test ID") +
+  xlab("Test ID") +
+  ylab("g emitted / g fuel burned") +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45))
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(benzene, aes(x= stove, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  ggtitle("Benzene EFs by Stove") +
+  xlab("Stove type") +
+  ylab("g emitted / g fuel burned") +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(benzene, aes(x= stove, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  facet_grid( ~ stovecat, scales = 'free') +
+  ggtitle("Benzene EFs by Stove Type") +
+  xlab("Stove type") +
+  ylab("g emitted / g fuel burned") +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(benzene, aes(x= stove, pol_fuel_mass_ratio, colour = stovecat)) +
+  geom_point() +
+  facet_grid( ~ fuelcat, scales = 'free') +
+  ggtitle("Benzene EFs by Fuel Type and Stove Type") +
+  xlab("Stove type") +
+  ylab("g emitted / g fuel burned") +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 45,  hjust = 0.9))
+```
+
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width = 10}
+ggplot(benzene, aes(x= id, pol_fuel_mass_ratio, colour = fuel)) +
+  geom_point() +
+  facet_wrap(fuelcat ~ stovecat) +
+  ggtitle("Benzene EFs by Stove and Fuel categories") +
```

Comment:
we should define a manual fuel color-scale so they always appear the same regardless of subsetting (Kelsey? 👍 )